### PR TITLE
Bypass screenshot restriction 

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+id: bypass-screenshot
+title: Bypass Screenshot Restriction
+sidebar_label: Bypass Screenshot Restriction
+hide_table_of_contents: true
+---
+
+If you test Android mobile apps on Sauce Labs real devices and you see a black screen in your live testing session, you might need to enable the **Bypass Screenshot Restriction**.
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:8]:
```suggestion
If you test Android mobile apps on Sauce Labs real devices and see a black screen in your live testing session, you might need to enable the **Bypass Screenshot Restriction**.
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Under the right conditions, when a user enables **Bypass Screenshot Restriction**, this allows Sauce Labs to work around the **FLAG_SECURE** setting so that the app can be tested. However, your ability to work around this setting is limited as described in [Workarounds for FLAG_SECURE Apps](#workarounds-for-flag_secure-apps).
+
+:::note
+Legacy app storage in Sauce Labs does not support this setting.
+:::
+
+The results of this setting vary based on the type of testing you’re doing:
+* Live - If you attempt to run a live test on a real device, you will not be able to interact with any screens that have this setting.
+* Automated - You can run automated tests on real devices but you will not be able to see video associated with those tests.
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:20]:
```suggestion
* Automated - You can run automated tests on real devices, but you will not be able to see video associated with those tests.
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## About FLAG_SECURE
+
+The **FLAG_SECURE** flag is intended to let app developers designate sensitive screens in applications as secure. This means the designated windows can’t appear in screenshots, streaming video, or other displays deemed insecure. The actual effects of this flag depends on the Android mobile OS version:
+* Android 7.0 and 8.0 - Video works but you can’t take a screenshot
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:26]:
```suggestion
* Android 7.0 and 8.0 - Video works, but you can’t take a screenshot
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* Android 9+ - You can’t see video or screenshots
+
+For more information, see [FLAG_SECURE](https://developer.android.com/reference/android/view/WindowManager.LayoutParams.html#FLAG_SECURE).
+
+## Workarounds for FLAG_SECURE Apps
+
+Your ability to work around this setting to test depends on a few factors:
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:33]:
```suggestion
Your ability to work around this setting to test will depend on a few factors:
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* Who owns the app:
+  * If your developers have created this app, they should be able to provide you with a version that does not have the **FLAG_SECURE** flag enabled.
+  * If your company has built this app on top of another company’s technology, your ability to request a version of the app without this flag is limited.
+* Whether **Bypass Screenshot Restriction** is an option:
+  * If your developers have created this app and they can’t provide you with an app that doesn’t use **FLAG_SECURE**, the **Bypass Screenshot Restriction** is an option.
+  * If you’re testing a scenario that includes interacting with an app owned by another company, **Bypass Screenshot Restriction** won’t work. For example if you need to use Google Pay to purchase something on your app, you won’t see video for that part of the purchase process.
+  * If your app is built on another company’s technology, **Bypass Screenshot Restriction** won’t work.
+
+:::note
+In cases where you’re using a third party app, you might be able to test your data directly on that app using live testing on a web browser.
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:43]:
```suggestion
In cases where you’re using a third party app, you might be able to test your data directly on that app using [live testing on a web browser](/web-apps/live-testing/live-cross-browser-testing/).
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+:::
+
+## Enable Bypass Screenshot Restriction
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:46]:
```suggestion

```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+* An understanding of the limitations of this option
+
+### Editing the App
+
+1. Go to **Live > Mobile App**.
+2. Hover over the app you want to test.
+3. Click **Settings**.
+4. Under **Default Settings > Instrumentation** on the right, find **Bypass Screenshot Restriction**.
+5. Click the slider to **Enabled**.
+6. Click **Back to App Selection** at the top to test your app.
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:62]: I'd move this to the top

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 8]
```diff
@@ -59,7 +59,7 @@ The app settings screen is only available for real device testing.
...
-| Bypass Screenshot Restriction <br/><p><span className="sauceDBlue">ANDROID ONLY</span></p> | Enable/disable Bypass Screenshot Restriction (not supported on apps uploaded to the legacy sauce storage). Enabling **Bypass Screenshot Restriction** allows you to take screenshots of your app during tests, even if your app does not allow screenshots for security reasons. |
+| Bypass Screenshot Restriction <br/><p><span className="sauceDBlue">ANDROID ONLY</span></p> | Enable/disable Bypass Screenshot Restriction (not supported on apps uploaded to the legacy sauce storage). If you test Android mobile apps on Sauce Labs and you see a black screen in your live testing session, you might need to enable the <b>Bypass Screenshot Restriction</b>. This allows Sauce Labs to work around a setting on those apps that prevents screenshots or videos from being taken. However, there are other details to keep in mind. To effectively test apps that have this setting, see [Bypass Screenshot Restriction](/mobile-apps/features/bypass-screenshot). |
```
Comment (kimsauce) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:62]:
```suggestion
| Bypass Screenshot Restriction <br/><p><span className="sauceDBlue">ANDROID ONLY</span></p> | Enable/disable Bypass Screenshot Restriction (not supported on apps uploaded to the legacy sauce storage). If you're testing Android mobile apps on Sauce Labs and see a black screen in your live testing session, you might need to enable the <b>Bypass Screenshot Restriction</b>. This allows Sauce Labs to work around a setting on those apps that prevents screenshots or videos from being taken. However, there are other details to keep in mind. To effectively test apps that have this setting, see [Bypass Screenshot Restriction](/mobile-apps/features/bypass-screenshot). |
```

File: docs/mobile-apps/features/bypass-screenshot.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  * If you’re testing a scenario that includes interacting with an app owned by another company, **Bypass Screenshot Restriction** won’t work. For example if you need to use Google Pay to purchase something on your app, you won’t see video for that part of the purchase process.
+  * If your app is built on another company’s technology, **Bypass Screenshot Restriction** won’t work.
+
+:::note
+In cases where you’re using a third party app, you might be able to test your data directly on that app using live testing on a web browser.
+:::
+
+## Enable Bypass Screenshot Restriction
+
+If you are testing your company’s proprietary app on a real device, you might be able to workaround the **FLAG_SECURE** setting by enabling **Bypass Screenshot Restriction** on that app under certain circumstances.
```
Comment (kimsauce) [inline docs/mobile-apps/features/bypass-screenshot.md:48]:
```suggestion
If you are testing your company’s proprietary app on a real device, you might be able to work around the **FLAG_SECURE** setting by enabling **Bypass Screenshot Restriction** on that app under certain circumstances.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Had a few suggestions!

**Review by lauramthomas — COMMENTED** [review]
Thanks for the feedback!

**Review by lauramthomas — COMMENTED** [review]
@kimsauce I made your changes a while back, but must have forgotten to tag you to approve the review. Oops.

**Comment by kimsauce** [comment]
@aoleszyn , this is the **FLAG_SECURE** doc you were waiting on -- it has been published! Can you please let Wendy know in the Jira ticket?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**